tools/release-ci/pkg/config/presubmit.go

```diff
@@ -25,12 +25,15 @@ type Presubmit struct {
 // GcsPath is the base path for job data. Pre-submit jobs
 // store their under the directory of their PR
 func (p *Presubmit) GcsPath() string {
-	return fmt.Sprintf("%s/%s%s/%d/%s/%d/", PullRequestPrefix, PullRequestIndex, repositoryPath(p.RepoOwner, p.RepoName), p.PullNumber, p.JobName, p.BuildNumber)
+	return fmt.Sprintf("%s/%s%s/%d/%s/%d%s/", PullRequestPrefix, PullRequestIndex, repositoryPath(p.RepoOwner, p.RepoName), p.PullNumber, p.JobName, p.BuildNumber, testPath(p.TestName))
 }
 
 // Aliases are the paths to symlink to the GcsPath.
 // Pre-submit jobs alias under the pull directory.
 func (p *Presubmit) Aliases() []string {
+	if len(p.TestName) > 0 {
```

Comment:
We've messed up separation of concerns here

Reply:
Will fix